htdocs/core/class/notify.class.php

```diff
@@ -390,7 +397,7 @@ function send($notifcode, $object, $filename_list=array(), $mimetype_list=array(
 							$outputlangs->setDefaultLang($obj->default_lang);
 						}
 
-						$subject = '['.$mysoc->name.'] '.$outputlangs->transnoentitiesnoconv("DolibarrNotification");
+						$subject = '['.$mysoc->name.']['.$proj->title.'] '.$outputlangs->transnoentitiesnoconv("DolibarrNotification");
```

Comment:
If object is not linked to a project, proj->title is not defined, even $proj is not defined